src/preact/base-element.js

```diff
@@ -369,9 +369,11 @@ export class PreactBaseElement extends AMP.BaseElement {
    * A callback called immediately after mutations have been observed on a
    * component. This differs from `checkPropsPostMutations` in that it is
    * called in all cases of mutation.
+   * @param {Array<MutationRecord>} records
    * @protected
    */
-  mutationObserverCallback() {}
+  // eslint-disable-next-line
```

Comment:
Let's remove the es-lint disable and rename `records` -> `unusedRecords`. A bit annoying, but that's what we usually do.

Reply:
Thanks, updated!